### ISSUE TITLE
Staging Review finding: Focus starts on segmented progress bar, then jumps to alert

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#115278]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Lifestage Pension and Burial
**Product:** Pension
**Feature:** Intent to File V1

## Findings details
**VA.gov Experience Standard - issue:** [User experiences predictable interactions with components and patterns.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @rsmithadhoc (Accessibility)

### Description

When the page loads, the focus starts on the segmented progress bar, but then jumps to the intent to file alert when that loads. This can be disorienting for screen reader, keyboard, or users with cognitive impairments.

<details><summary>Link, screenshot or steps to recreate</summary>

https://staging.va.gov/pension/apply-for-veteran-pension-form-21p-527ez/applicant/information

![Image](https://github.com/user-attachments/assets/86afc998-613f-4b2d-8a1c-47093406ed82)

</details>

## Recommended action

Use [ARIA live regions for alerts](https://design.va.gov/components/alert/#assign-an-appropriate-aria-role) so screen readers announce the message. Reserve focus changing for situations that require a user action, such as a modal.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 2.4.3 Focus Order - Level A](https://www.w3.org/TR/WCAG22/#focus-order)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by skylerschain** [comment]
@ataker  You want to help me with this one?

**Comment by skylerschain** [comment]
That makes sense to me. Also, since this is Step 1 of the form, a screen reader user will mostly likely navigate through the alert as part of consuming the content of the page. So it seems to match nicely the order of priority of the UI itself.
